libc-test/build.rs

```diff
@@ -2509,6 +2509,13 @@ fn test_linux(target: &str) {
         _ => false,
     });
 
+    cfg.skip_field(move |struct_, field| {
+        struct_ == "tcp_info"
+            && (field == "tcpi_snd_wscale"
+                || field == "tcpi_rcv_wscale"
+                || field == "tcpi_delivery_rate_app_limited")
+    });
```

Comment:
If the Rust struct does not contain these fields or if these fields are private, this should not be necessary.

Reply:
This is breaking all tests because it is replacing the closure above (previous `cfg.skip_fields` call) with this one. You need to add `tcp_info` as a case for the previous closure and not as a new closure here.